modules/sugar/src/main/ts/ephox/sugar/api/events/Types.ts

```diff
@@ -1,6 +1,7 @@
+import { Event } from '@ephox/dom-globals';
 import Element from '../node/Element';
 
-export interface EventArgs<T = any> { // Set to any since there might be a lot of code working directly with the sub types of Event
+export interface EventArgs<T = Event> {
```

Comment:
I couldn't find any places where this assumed it was a sub type, so I've changed this to default to `Event` instead of `any`. I also made the other modules that use this (eg `DomEvent`) make use of the actual event types now.